packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/webview_flutter_test.dart

```diff
@@ -1170,9 +1170,6 @@ Future<void> main() async {
 // JavaScript booleans evaluate to different string values on Android and iOS.
 // This utility method returns the string boolean value of the current platform.
 String _webviewBool(bool value) {
-  if (defaultTargetPlatform == TargetPlatform.iOS) {
```

Comment:
As per discussion: https://github.com/flutter/plugins/pull/4990#discussion_r833510623

The `runJavascript` methods will properly print out boolean rather than converting them to ints.

Reply:
It is. But I thought your comment was implying that this behavior should only be changed for `evaluateJavascript`. 

Could it not be considered a bug for `runJavascript` and `runJavascriptReturningResult` since it is returning the wrong type?

Although, v4 is approaching so I actually agree that we should avoid breaking for now. I decided to change it back and added a test for `runJavascriptReturningResult`.

Reply:
I can see it both ways here; I was looking at it as not being obviously *wrong* like the null case in the newer APIs.

We should file an issue with the label for being s breaking change, and reference it with a to-do, so we fix it in V4.